test/track.js

```diff
@@ -1,6 +1,6 @@
 var proxyquire = require('proxyquire'),
     Sinon      = require('sinon'),
-    http       = require('http'),
+    https      = require('https'),
```

Comment:
I'm not seeing the actual change to the product code to switch from http -> https, is that done in another PR ?